src/main/java/seedu/address/model/score/exceptions/BadDateException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.score.exceptions;
+
+/**
+ * Signals that the operation is unable to parse the date due to incorrect format.
+ */
+public class BadDateException extends RuntimeException {
+    public BadDateException() {
+        super("Date is not in the correct format!");
+    }
```

Comment:
Maybe can add one more sentence that to show the correct format.

Reply:
I have made use of the checkArgument method to validate our models.